x-pack/plugin/core/src/main/java/org/elasticsearch/license/XPackLicenseState.java

```diff
@@ -400,9 +404,13 @@ private static boolean isBasic(OperationMode mode) {
         /** True if the license is active, or false if it is expired. */
         final boolean active;
 
-        Status(OperationMode mode, boolean active) {
+        /** The current expiration date of the license; Long.MAX_VALUE if not avILble yet. */
```

Comment:
s/avILble/available. Nit: also change the description of the Status class above to add the expiration date